src/components/ListItem.jsx

```diff
@@ -49,14 +50,16 @@ export function ListItem({
 
 	return (
 		<li className="ListItem">
-			<label>
+			<label className="ListItem-label">
 				<input
+					className="ListItem-checkbox"
 					type="checkbox"
 					name="checkbox"
 					checked={isChecked}
 					onChange={handleCheck}
 				/>
-				{name}
+				<span className="ListItem-name">{name}</span>
+				<code className="ListItem-urgency">{urgency}</code>
```

Comment:
We tested it on an accessibility site, and it seemed to work well. Ismarji and I talked, and we decided to leave it for now, since we'll be working on styling soon.